File: Java/Node/src/Node/Biz/Interfaces/Solicit/IPostProcess.java
package Node.Biz.Interfaces.Solicit;

import java.rmi.RemoteException;

import Node.Biz.Custom.PostParam;
/**
 * <p>This class create Solicit Post Process Interface.</p>
 * <p>Company: enfoTech & Consulting, Inc.</p>
 * @author enfoTech
 * @version 2.0
 */


public interface IPostProcess {
	  /**
	   * Execute
	   * @param token The token
	   * @param returnURL The return URL or the data flow name
	   * @param request The request
	   * @param parameters The input parameters
	   * @param param PostParam object
	   * @throws RemoteException
	   * @return PostParam object
	   */
  public PostParam Execute (String token, String returnURL, String request, Object[] params, PostParam param) throws RemoteException;
}

File: DotNet/Node.Core/Biz/Interfaces/NodePing/IPostProcess.cs
using System;

using Node.Core.Biz.Manageable.Parameters;

namespace Node.Core.Biz.Interfaces.NodePing
{
    /// <summary>
    /// The Interface for NodePing PostProcess
    /// </summary>
    public interface IPostProcess
    {
        /// <summary>
        /// PostProcess for NodePing.
        /// </summary>
        /// <param name="hello">Input string</param>
        /// <param name="param">Operation related parameters</param>
        /// <returns>Input string </returns>
        void Execute(string hello, PostParam param);
    }
}
